### ISSUE TITLE
Lucius (218640) Won't launch with D9vk

### ISSUE THREAD

**Issue by FurtadoPires** [ValveSoftware/Proton#3063]
# Compatibility Report
- Name of the game: Lucius
- Steam AppID: 218640

## System Information
[Machine info](https://gist.github.com/FurtadoPires/331ba48eaf4ef82fff6c15dee2691904)
- Proton version: 4.11.4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Game launch normally with no launch parameters, but won't launch when I use PROTON_USE_D9VK=1 %command%

[Proton log](https://gist.github.com/FurtadoPires/856f2f07ccfb84d2e8f13723d22d7401)

**Comment by aeikum** [comment]
Please report this on the [D9VK](https://github.com/Joshua-Ashton/d9vk) github.

**Comment by lucifertdark** [comment]
The game is running with Proton 4.11-12
[steam-218640-4.11-12.log](https://github.com/ValveSoftware/Proton/files/4209597/steam-218640-4.11-12.log)
But instant crash on Proton 5.0-2
[steam-218640-5.0-2.log](https://github.com/ValveSoftware/Proton/files/4209598/steam-218640-5.0-2.log)
system specs https://gist.github.com/lucifertdark/584c14275612e8e3c4a8070d428d8a02

**Comment by soredake** [comment]
Crashes with 5.0-2.

[steam-218640.log](https://github.com/ValveSoftware/Proton/files/4233214/steam-218640.log)

**Comment by FurtadoPires** [comment]
Game is probably crashing with Proton 5 due to D9VK being activated by default.

@aeikum is it now valid to post here about D9VK problems?

**Comment by aeikum** [comment]
@FurtadoPires Yes. If it works in wined3d but fails with DXVK, it can also be useful to report them to DXVK directly.

**Comment by FurtadoPires** [comment]
Tested again with Proton 5.13.1

The game now lauches, but crash after the cut scene.

`err:   DxvkMemoryAllocator: Mapping memory failed with VK_ERROR_MEMORY_MAP_FAILED
err:   DxvkMemoryAllocator: Memory allocation failed
err:     Size:      4102400
err:     Alignment: 16
err:     Mem flags: 0x6
err:     Mem types: 0xf
err:   Heap 0: 1465 MB allocated, 1409 MB used, 1490 MB allocated (driver), 1769 MB budget (driver), 1792 MB total
err:   Heap 1: 1872 MB allocated, 1848 MB used, 1922 MB allocated (driver), 3237 MB budget (driver), 3072 MB total
err:   Heap 2: 16 MB allocated, 0 MB used, 22 MB allocated (driver), 250 MB budget (driver), 256 MB total
1780.385:00cc:00d0:trace:seh:MSVCRT_raise (22)
1780.386:00cc:00d0:trace:seh:NtSetInformationThread (0xfffffffe,10,0x31e954,4)`

[Complete log](https://gist.github.com/FurtadoPires/a3c4a43e632c9c2d3d85948646307b5d)
